introspection-engine/connectors/mongodb-introspection-connector/src/sampler/statistics.rs

```diff
@@ -115,7 +115,7 @@ impl Statistics {
             };
 
             let documentation = if percentages.has_type_variety() {
-                Some(format!("{}", percentages))
+                Some(format!("Multiple data types found {}", percentages))
```

Comment:
Late to the party, but would be good to add a `:` or `;` in there.

Reply:
done in https://github.com/prisma/prisma-engines/pull/2320/